src/hotel-management-app/src/app/customer/payment/payment.component.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<mat-toolbar color="primary">
+    <span>Payment</span>
+</mat-toolbar>
+
+<div class="payment-form">
+    <mat-form-field>
+        <mat-label>Cardholder Name</mat-label>
+        <input matInput [(ngModel)]="paymentDetails.cardholderName">
```

Comment:
Could you add validations to this inputs

Reply:
resolved